### PR TITLE
Add release blog for Podman 1.5.0

### PR DESCRIPTION
**Pull request by mheon**
As the title says

### CODE REVIEW

File: _posts/2019-08-14-podman-release-v1.5.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+## Podman has gone 1.5!
+
+Podman 1.5.0 has been released! We’ve made major improvements to `podman exec`, `podman generate kube`, and rootless containers in this release. Stability has also been a focus, and we’ve fixed over 30 bugs and several performance issues. The new 1.5.0 release is available for Fedora and Ubuntu right now!
+
+<!--readmore-->
+
+With this new release, Podman has picked up a number of improvements to core container functionality. The `podman exec` command has been completely reworked, including improved handling for attaching to containers. Expect to see more work on `exec` in future releases. CGroups have also seen major work, with support for CGroup namespaces via the `--cgroupns` flag to `podman create` and `podman run`, and support for CGroups v2 when using the `crun` OCI runtime (more details [here](https://www.scrivano.org/2019/05/12/rootless-resources-management-with-podman-on-fedora-30/). The `podman generate kube` command has also been improved and now includes volumes mounted into containers. Finally, we’ve addressed several memory leaks and other performance issues, and Podman should be much more responsive on systems under high load.
```
Comment (TomSweeneyRedHat) [inline _posts/2019-08-14-podman-release-v1.5.0.md]: "(more details here" is missing an end paren.  Either add one, or I think I'd prefer dropping the starting paren and putting a comma after "OCI runtime".

File: _posts/2019-08-14-podman-release-v1.5.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Rootless containers have also been improved, featuring improved handling for privileged containers and the ability to use container health checks. Podman now has experimental support for running rootless containers with a single UID and GID using the new `ignore_chown_errors` storage option. This allows Podman to be run without the `newuidmap` and `newgidmap` binaries, and removes the need for any elevated privileges to start rootless containers. This approach is more limited (but more secure) than normal rootless containers.
+
+As always, please visit our release notes on [GitHub](https://github.com/containers/libpod/blob/master/RELEASE_NOTES.md) to see the full changelog.
+
+You can find instructions for installing Podman [here](https://github.com/containers/libpod/blob/master/install.md)
```
Comment (TomSweeneyRedHat) [inline _posts/2019-08-14-podman-release-v1.5.0.md]: I'll get an ending period out of you this time!

**Comment by TomSweeneyRedHat** [comment]
LGTM, if you could also add a "new" post similar to `_posts/2019-05-10-new.md` that would be great, otherwise I'll circle back to do so.

**Comment by TomSweeneyRedHat** [comment]
Should have been "LGTM other than the nits".

**Comment by mheon** [comment]
Comments addressed, aside from the New post

**Pull request merged**